chain/db/table.go

```diff
@@ -46,3 +46,10 @@ type DataSourceRevision struct {
 	BlockHeight    int64  `gorm:"not null"`
 	TxHash         []byte `gorm:"not null"`
 }
+
+type Block struct {
+	Height    int64  `gorm:"primary_key;auto_increment:false"`
+	Timestamp int64  `gorm:"not null"`
+	Proposer  string `gorm:"not null"`
+	BlockHash string `gorm:"unique;not null"`
```

Comment:
I don't think we need to say it's unique?

Reply:
We can store block hash as []byte